File: gymassistantv2.client/src/app/domain/TrainingSetExercise.ts
import { Exercise } from "./Exercise";
import { TrainingSet } from "./TrainingSet";

export class trainingSetExercise{
    public id:  number;
    public exercise: Exercise;
    public trainingSets: TrainingSet[];


    constructor(exercise: Exercise, trainingSets: TrainingSet[] ){
        this.exercise = exercise;
        this.trainingSets = trainingSets;
}
}

File: DataAccess/Entities/TrainingSetExercise.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccess.Entities
{
    public class TrainingSetExercise: BaseEntity
    {
        public Exercise Exercise { get; set; }
        public List<TrainingSet> TrainingSets { get; set; }

    }
}
